ssh/session.go

```diff
@@ -24,7 +24,7 @@ import (
 
 var (
 	ErrInvalidSessionTarget = errors.New("invalid session target")
-	ErrBillingBlock         = errors.New("reached the device limit, update to premium or choose up to 3 devices")
+	ErrBillingBlock         = errors.New("you reached the device limit, you need to update the premium plan or choose up to 3 devices")
```

Comment:
```suggestion
	ErrBillingBlock         = errors.New("You reached the free device limit, please upgrade to the premium plan or choose up to 3 devices.")
```

Reply:
@otavio golang lint doesn't allow you to add capital letters or dot. 
What to do in this case?